### PR TITLE
Dolphyn viz tools

### PR DESCRIPTION
**Pull request by shakesy94**

### CODE REVIEW

File: viz_tools/src/data_wrangling_long_v1.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    fig.savefig(f"{directory}/{filename}.png", bbox_inches='tight', dpi=300)
+    plt.close(fig)  # Close the figure
+
+
+def find_latest_result_folder(path):
+    result_folders = [folder for folder in os.listdir(path) if folder.startswith("Results")]
+    if not result_folders:
+        raise ValueError("No 'Results' folders found.")
+        
+    result_numbers = [int(folder.split("_")[-1]) if "_" in folder else 0 for folder in result_folders]
```
Comment (bennettm) [inline viz_tools/src/data_wrangling_long_v1.py]: This line of code does not consider an existing folder: Results_Example. Should do a regular expression matching or filtering.

File: viz_tools/src/data_wrangling_long_v1.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    path = path + '/' + latest_result_path + '/'
+    return(path)
+    
+def open_results_file(file_name, run):
+    run = runs_directory_path + '/' + run
+    path = latest_result_finder(run)
+    if file_name.startswith('HSC'):
+        path = path + 'Results_HSC/'
+    path = path + file_name
+    #print(path)
```
Comment (bennettm) [inline viz_tools/src/data_wrangling_long_v1.py]: This line of commented code can be removed.

File: src/viz_tools/src/viz/create_viz.py [hunk 3]
```diff
@@ -1,9 +1,34 @@
 import pandas as pd
 import matplotlib.pyplot as plt
 import os
-import yaml
-from generators_color_dict.py import elec_colors
-from generators_color_dict.py import h2_colors
+#import yaml
+#from generators_color_dict.py import elec_colors
+#from generators_color_dict.py import h2_colors
```
Comment (bennettm) [inline src/viz_tools/src/viz/create_viz.py:6]: Clean up these three lines of commented out codes.

File: src/viz_tools/src/data_wrangling/data_wrangling_long.py
Comment (bennettm) [inline src/viz_tools/src/data_wrangling/data_wrangling_long.py:1]: Remove this commented line:
def open_results_file(file_name, run):
    #run = runs_directory_path + '/' + run

File: Example_Systems/SmallNewEngland/ThreeZones_Liquid/Figures/elec_generation.jpeg
Comment (bennettm) [inline Example_Systems/SmallNewEngland/ThreeZones_Liquid/Figures/elec_generation.jpeg:1]: It does not have colors for this graph.

Comment (bennettm) [inline Example_Systems/SmallNewEngland/ThreeZones_Liquid/Figures/elec_generation.jpeg:1]: I have found there are other graphs that don't have colors as well. Shouldn't these have colors?

File: src/viz_tools/src/viz/create_viz.py (hunk 3 above)
Comment (shakesy94) [reply src/viz_tools/src/viz/create_viz.py]: resolved

**Review by shakesy94 — COMMENTED** [review]
responded to mary's comments

File: docs/src/global_data_documentation.md [hunk 4]
```diff
@@ -23,7 +23,11 @@
 |**Miscellaneous**|
 |PrintModel | Flag for printnig the model equations as .lp file.|
 ||1= including the model equation as an output|
-||0 for the model equation not being included as an output|
+||0= for the model equation not being included as an output|
+
+|VisualizeData | lot basic data visualizations for quick diagnostics (call_dolphyn_viz)|
```
Comment (bennettm) [inline docs/src/global_data_documentation.md:28]: Is it "Plot basic data visualizations..." here, @shakesy94 ?

File: src/viz_tools/example_viz/S1_Jul_13_2040_90_40EC_Base_SF_0.2_co2_0.4electro_opt_co2stor_baseline_sf_0.0/Run.jl~
Comment (bennettm) [inline src/viz_tools/example_viz/S1_Jul_13_2040_90_40EC_Base_SF_0.2_co2_0.4electro_opt_co2stor_baseline_sf_0.0/Run.jl~:1]: @shakesy94 : This Run.jl~ somehow got committed here.  This file with ~ is a temp file.  Need to remove this file.

**Review by bennettm — APPROVED** [review]
I left two comments and made those fixes so I will approve this pull request.

**Comment by shakesy94** [comment]
Hey @Betristor ! I am having an issue with this PR where it fails a check because numpy is not in the package dependencies. I think what is going on, is that my python script imports numpy. Julia does not like that it is not installed/in DOLPHYN’s dependencies. However, if you install Conda, the miniconda env should include numpy so you do not need to install it separately. This has not caused me any issues when I run it on my machine even though I never explicitly installed numpy. Would you know why this could be failing?

**Pull request closed without merging**